contrib/cirrus/find_unicode_control2.py

```diff
@@ -0,0 +1,199 @@
+#!/usr/bin/env python3
+"""Find unicode control characters in source files
+
+By default the script takes one or more files or directories and looks for
+unicode control characters in all text files.  To narrow down the files, provide
+a config file with the -c command line, defining a scan_exclude list, which
+should be a list of regular expressions matching paths to exclude from the scan.
+
+There is a second mode enabled with -p which when set to 'all', prints all
+control characters and when set to 'bidi', prints only the 9 bidirectional
+control characters.
+"""
+from __future__ import print_function
+
+import sys, os, argparse, re, unicodedata, subprocess
+import importlib
+from stat import *
+
+def _unicode(line, encoding):
+    if isinstance(line, str):
+        return line
+    return line.decode(encoding)
+
+import platform
+if platform.python_version()[0] == '2':
```

Comment:
Again, we know python version is >= 3 from shebang line.

Reply:
Not my code...and I wish there was a URL I could point at in a comment.  Sadly there isn't, this was downloaded directly from the advisory mentioned :cry: